api/users/controller.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable func-names */
+/* eslint-disable no-confusing-arrow */
+const User = require('./model');
+
+exports.userCreate = function (req, res, next) {
+  const user = new User({
```

Comment:
right now, I am sanitizing input data from req.body in a separate file. I'm passing in an object to this controller file for userCreate. So we can change this to something like:

```javascript
exports.userCreate = function (info) {
  const user = new User(info)
```

for ref, this is the post to create a new user:

```javascript
  .post(validate.signup, sanitize.user, (req, res) => {
    controller
      .create(req.user)
      .then(savedUser => send(res, 201, savedUser))
      .catch(err =>
        send(res, 500, { err, message: `server failed to save new user` }),
      );
  });
```

Reply:
here is the link to the route handler file in auth branch `https://github.com/Lambda-School-Labs/pic_me/blob/auth/api/endpoints/user.js#L24`

Reply:
and if you disagree with only having the route handler send responses, please comment as well. we can discuss design changes for the BE!